hack/numaflow-controller-def-generator/kustomization.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+commonLabels:
+  app.kubernetes.io/instance: '{{ .InstanceID }}'
+nameSuffix: "{{ .InstanceSuffix }}"
```

Comment:
wow, so other than the `instance` field in the `numaflow-controller-config`, these lines took care of everything - nice job!

Reply:
thanks! yes, those were some nice findings